workloads/examples/k8s/deep-learning-examples/docker-compose.yaml

```diff
@@ -0,0 +1,246 @@
+services:
+  cuda-optimized-fastspeech:
+    image: registry.local/cuda-optimized-fastspeech
```

Comment:
**Optional nit**:

Can `docker-compose.yaml` files be templated?

If so: please make it possible to specify an alternate registry instead of `registry.local`

If not: maybe just add a note to the doc to specify how to use an alternate docker-compose file, and specify your own registry?